File: YTMSiparis/KullanıcıUyg/python/Kullanici.py
class Kullanici:
    def __init__(self, isim=None, mail=None, adres=None):
        self._isim = isim
        self._mail = mail
        self._adres = adres

    @property
    def isim(self):
        return self._isim

    @isim.setter
    def isim(self, value):
        self._isim = value

    @property
    def mail(self):
        return self._mail

    @mail.setter
    def mail(self, value):
        self._mail = value

    @property
    def adres(self):
        return self._adres

    @adres.setter
    def adres(self, value):
        self._adres = value

    def adres_guncelle(self, yeni_adres):
        self.adres = yeni_adres

    def giris_yap(self):
        print("Giriş Yapıldı.")

    def __str__(self):
        return f"{self.isim} {self.mail} {self.adres}"

File: YTMSiparis/KullanıcıUyg/c#/SRP/SRP/Kullanici.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SRP
{
    public class Kullanici
    {
        private string isim {  get; set; }
        private string mail { get; set; }
        private Adres adres;
        public string Isim
        {
            get { return isim; }
            set { isim = value; }
        }

        public string Mail
        {
            get { return mail; }
            set { mail = value; }
        }

        public Adres Adres
        {
            get { return adres; }
            set { adres = value; }
        }
        public void adresGuncelle(Adres adres)
        {
            this.adres = adres;
        }
        public string getAdres()
        {
            return adres.ToString();
        }
        public override string ToString()
        {
            return $"{isim} {mail} {getAdres()}";
        }
        public void girisYap()
        {
            Console.WriteLine("Giriş Yapıldı.");
        }
    }
}
